setup.cfg

```diff
@@ -11,7 +11,11 @@ statistics = True
 
 [flake8]
 max-line-length = 120
-ignore = R903, F401, F403
+ignore = R903
+per-file-ignores =
```

Comment:
I thought the issue was related with the backends but the files are others in this case I prefer disable that per line as I mentioned in the previous comment for 401